gazelle/python/generate.go

```diff
@@ -54,6 +55,24 @@ func GetActualKindName(kind string, args language.GenerateArgs) string {
 	return kind
 }
 
+func makeCompiledGlobs(globs []string) []glob.Glob {
+	compiledGlobs := []glob.Glob{}
+	for _, value := range globs {
+		compiledGlob := glob.MustCompile(value)
+		compiledGlobs = append(compiledGlobs, compiledGlob)
+	}
+	return compiledGlobs
+}
+
+func matchesAnyGlob(s string, globs []glob.Glob) bool {
+	for _, g := range globs {
+		if g.Match(s) {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Is there a better way to do this in go?

In python, these would be replaced by something like:

```python
compiled_globs = [MustCompile(s) for s in string.split(",")]
...
else if any(glob.match(f) for glob in compiled_globs):
    pyTestFilenames.Add(f)
```

Reply:
This is good enough. Slightly verbose, but easy to understand.